drivers/client_access_control/app/controllers/client_access_control/history_controller.rb

```diff
@@ -117,7 +117,7 @@ def set_dates
         includes(:service_history_services, :organization).
         each do |enrollment|
           project_type = enrollment.send(enrollment.class.project_type_column)
-          project_name = name_for_project(enrollment.project_name)
+          project_name = enrollment.project&.name(current_user)
```

Comment:
This is slower because we are accessing the `project` instead of inferring confidentiality from `she.project_name` field. I think its worth the extra lookup?